src/components/TipInput.vue

```diff
@@ -13,14 +13,22 @@
           v-if="!tipUrlStats.tokenTotalAmount.length || +tipUrlStats.totalAmountAe !== 0"
           :amount="tipUrlStats.totalAmountAe"
         />
+      </template>
+    </Component>
+    <Dropdown
+      v-if="!userAddress && tipUrlStats && tipUrlStats.tokenTotalAmount.length"
+      :options="tipUrlStats.tokenTotalAmount"
+      show-right
+      class="token-tips"
+    >
+      <template v-slot="{ option }">
```

Comment:
I have checked it locally and seems that you don't need an extra template:
```suggestion
      v-slot="{ option }"
    >
```